data/remnant/remnant 2 cognizance.txt

```diff
@@ -858,7 +868,8 @@ mission "Remnant: Cognizance 21"
 			`	Dusk leans over the holo, peering at the details. "The mission is a very standard arc: We go to the station, set up our equipment to get some detailed readings, and return here once we are done. For these initial readings we should need about a day onsite, and we will leave some of our equipment to take ongoing recordings for retrieval later. After that... Well, we will see what the data says and decide from there."`
 				accept
 	on accept
-		set "dusk busy"
+		set "Remnant: Dusk Busy"
+		"remnant met dusk" ++
```

Comment:
Why not just use below?
```suggestion
		set "remnant met dusk"
```
Granted, I'm not a wiz with using triggers. Was there something specific your method achieves that my suggestion doesn't in regards to using this in #6153 as well?

Reply:
The main difference here is that by using ++ it will increment upwards instead of just setting it. So it can be used to see if the player has been through more than one of these meeting later on. I do intend to make use of that later. So it is a case of "enables future branching without incurring any additional cost now"